src/contentScript/contentScript.tsx

```diff
@@ -5,9 +5,4 @@
 
 import { ButtonInjectorFactory } from "./buttonInjector/ButtonInjectorFactory";
 
-try {
-    ButtonInjectorFactory.getButtonInjector()?.inject();
-} catch (e) {
-    const message = e instanceof Error ? e.message : e;
-    console.log(`try-in-web-ide-browser-extension error: ${message}`);
-}
```

Comment:
This try catch did not work in the first place because the `inject()` call is an async call